### PR TITLE
CLB health monitor APIs implemented

### PR DESCRIPTION
**Pull request by manishtomar**
Implementation of https://developer.rackspace.com/docs/cloud-load-balancers/v1/developer-guide/#document-api-operations/monitors . This is needed by autoscale to test implementation of https://github.com/rackerlabs/otter/issues/1870.

### CODE REVIEW

File: mimic/test/test_loadbalancer.py [hunk 1]
```diff
@@ -472,6 +472,113 @@ def test_same_tenant_different_regions(self):
...
+        """
+        lb_id = self._create_loadbalancer()
+        d = request_with_content(
+            self, self.root, b"GET",
+            "{}/loadbalancers/{}/healthmonitor".format(self.uri, lb_id))
+        resp, body = self.successResultOf(d)
+        self.assertEqual(resp.code, 200)
+        self.assertEqual(json.loads(body.decode()), {"healthMonitor": {}})
+
+    def _test_health_monitor_404(self, method, req_body=b""):
```
Comment (BenjamenMeyer) [inline mimic/test/test_loadbalancer.py]: docstring please

**Comment by manishtomar** [comment]
Does this need more work for merging?

**Comment by glyph** [comment]
@manishtomar - sorry, will probably have to wait for next week :-\

**Comment by manishtomar** [comment]
@glyph I understand :). I was asking since @BenjamenMeyer had reviewed the code. Was wondering if he had any more comments.

**Comment by glyph** [comment]
@manishtomar oh, awesome! @BenjamenMeyer  - thanks for stepping up, great to have some other reviewers active on Mimic again :)

**Comment by BenjamenMeyer** [comment]
@manishtomar I'm good so :+1: from me.

**Pull request merged**